### PR TITLE
NF DEI interview call

### PR DESCRIPTION
**Pull request by IAlibay**
This is related to the outreach call text we sent around a couple weeks ago.

### CODE REVIEW

**Review by richardjgowers — APPROVED** [review]

**Review by orbeckst — APPROVED** [review]
lgtm

**Comment by orbeckst** [comment]
@IAlibay @fiona-naughton do you want to wait with publishing the post until you get an ok from everyone that you assigned?

I'll let you decide when to merge.

**Comment by IAlibay** [comment]
> @IAlibay @fiona-naughton do you want to wait with publishing the post until you get an ok from everyone that you assigned?
> 
> I'll let you decide when to merge.

Nah this should be enough, I'd prefer move quickly with this. Thanks @orbeckst @richardjgowers !

**Pull request merged**
